src/workerd/api/rtti.c++

```diff
@@ -116,6 +116,15 @@ struct EncoderModuleRegistryImpl {
     }
   }
 
+  template <typename Func>
+  void addBuiltinBundleFiltered(jsg::Bundle::Reader bundle, Func filter) {
+    for (auto module: bundle.getModules()) {
+      if (filter(module)) {
+        addBuiltinModule(module);
+      }
+    }
+  }
```

Comment:
Is it intentional that there are two copies of this function? The other is in `modules.h`.

Reply:
Yes. Two different class impls

Reply:
Fair nuff but I'm not sure what calls this one because it's in a .c++ file so the caller would need to be there?

Reply:
This is part of the type generation mechanism.